README.md

```diff
@@ -1,17 +1,17 @@
 <div align="center">
 
-  ![Logo White](https://raw.githubusercontent.com/Ekumen-OS/andino/humble/docs/logo_white.svg#gh-dark-mode-only)
+  ![Logo White](./docs/logo_white.svg#gh-dark-mode-only)
```

Comment:
@jballoffet I feel that you were using `...raw.githubusercontent...`  for a reason I don't know about.

Reply:
If you managed to make it work with a relative link, awesome! No other particular reason.